examples/active_fire_detection/README.md

```diff
@@ -0,0 +1,205 @@
+
+**Unpacking and restructuring**
+Now that the Google Drive downloads have unzipped, we can `oxen add`, `oxen commit`, and then `oxen push` them to our repository, first in their original format...
+![](images/oxenhub_pre_reorg.png)
+...then, after a bit of friction, in a reorganized format that will set us up for clearer delineation and linkage between our data and labels: 
+
+![](images/oxenhub_post_reorg.png)
+
+
+#### 2. Improving data-label linking and training ergonomics 
+In the source dataset, the original satellite image for a patch, the authors' manually annotated labels, and the 5 sets of algorithmically determined labels are linked together only implicitly through commonalities in the filenames:
+
+```
+Input data (LANDSAT) path: 
+- LC08_L1GT_226074_20200921_20200921_01_RT_p00811
+
+Manual annotation path: 
+- LC08_L1GT_226074_20200921_20200921_01_RT_v1_p00811
+
+Kumar-Roy: 
+- LC08_L1GT_226074_20200921_20200921_01_RT_Kumar-Roy_p00811
+
+Murphy: 
+- LC08_L1GT_226074_20200921_20200921_01_RT_Murphy_p00811
+
+Intersection:
+- LC08_L1GT_226074_20200921_20200921_01_RT_Intersection_p00811
+
+
+```
+
+While this isn't particularly difficult to parse with regular expressions at training or inference time, it's also an opportune hiding spot for sneaky human error and unnecessary data munging overhead on each of the individual engineers working on this project. 
+
+Further, while there are 9,044 total patches of raw imagery included in this dataset, the various annotation types vary in number and don't cover the same data: 
+- Manual annotations: 100
+- Kumar method: 391
+- Murphy method: 164
+- Schroeder method: 227
+- Intersection method: 118
+- Voting method: 198
+
+...so solidifying awareness of coverage across the various labeling types in one place is key to effective collaboration on this dataset. 
+
+**Creating a data -> label mapping file and validating with OxenHub**
+
+We parsed the filenames to the common patch_id, creating the following mapping file structure...
+
+
+![A new data-to-labels mapper file where one row = one original LANDSAT scene, with columns for each of the 6 label masks and their paths](images/oxenhub_good_organization.png)
+...associating each LANDSAT patch to the various label paths for that patch, when they exist. 
+
+We can use schema metadata to let OxenHub know which columns are relative filepaths:
+
+```
+TODO: Schema setting
+```
+
+which quickly elucidates an error in our processing. 
+
+**TODO IMAGE OF FAILED FILE EXISTENCE CHECK IN HUB**
+
+After hunting down the bug in our filepath processing code, we make a new commit with the fix and are ready to train! 
+
+**TODO IMAGE OF SUCCESSFUL FILE EXISTENCE CHECKS IN HUB**
+
+#### 3. Improving Geographic Awareness 
+##### Why emphasize geography?
```

Comment:
1. Does this make sense? It's a very EO-specific concept but obviously related to ML as a whole...

2. Is it too much? I needed something to ground the geographical additions, and it's super pertinent to this dset, but it's also a pet fav topic of mine so don't want to ramble

Reply:
This feels like too much, could be a whole post on it's own. I think this post should focus on migrating from GitHub / Google Drive / Zip Files to Oxen, and why that is so much better.